Scripts/NuGet/Shared/LICENSE.txt

```diff
@@ -0,0 +1,176 @@
+                                 Apache License
```

Comment:
i disagree with this change

Reply:
Acknowledged. I have not found an acceptable workaround.